quilt/tools/command.py

```diff
@@ -721,11 +737,94 @@ def install(package, hash=None, version=None, tag=None, force=False):
 
     pkgobj.save_contents()
 
+def _setup_env(env, files):
+    """ process data distribution. """
+    # TODO: build.yml is not saved in the package system, so re-load it here
+    environments = None
+    with open('build.yml') as fd:
+        buildfile = next(yaml.load_all(fd), None)
+        environments = buildfile.get('environments')
```

Comment:
Use `buildfile.get('environments', {})` - then you don't have to check for `None`. Actually, may want to check that it's a `dict`

Reply:
DONE